.github/workflows/CI.yml

```diff
@@ -87,7 +87,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: "recursive"
-      - run: python3 generator.py --yes
+      - run: python3 ./gir/generator.py --yes ../
```

Comment:
We created a symlink in the root of `gstreamer-rs` but this works just as well :)

Reply:
Ah that's a much better idea actually :) Thanks!